shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -421,6 +423,120 @@ public Intent build(@NonNull Context context) {
     }
   }
 
+  /**
+   * Creates a {@link NewEngineInGroupIntentBuilder}, which can be used to configure an {@link
+   * Intent} to launch a {@code FlutterActivity} that internally uses an existing {@link
+   * io.flutter.embedding.engine.FlutterEngineGroup} that is cached in {@link
```

Comment:
that is cached in **a** FlutterEngineGroup

Reply:
yes,  when we use this method. first, we should create a FlutterEngineGroup and cached in FlutterEngineGroupCache by flutterEngineGroupId. then we can use FlutterActivity.withNewEngineInGroup(flutterEngineGroupId) to create intent and start FlutterActivity